doc/source/security/encrypting-doublewrite-buffers.rst

```diff
@@ -0,0 +1,34 @@
+.. _encrypting-doublewrite-buffers:
+
+=======================================================================
+Encrypting Doublewrite Buffers
+=======================================================================
+
+The two types of doublewrite buffers used in |Percona Server| are encrypted differently.
+
+When the InnoDB system tablespace is encrypted, the ``doublewrite buffer`` pages
+are encrypted as well. The key which was used to encrypt the InnoDB system
+tablespace is also used to encrypt the doublewrite buffer.
+
+|Percona Server| encrypts the ``parallel doublewrite buffer`` with the respective
+tablespace keys. Only encrypted tablespace pages are written as encrypted in the
+parallel doublewrite buffer. Unencrypted tablespace pages will be written as
+unencrypted.
+
+.. important::
+
+    A server instance initialized with the encrypted InnoDB system tablespace
+    cannot be downgraded. It is not possible to parse encrypted InnoDB system
+    tablespace pages in a version of |Percona Server| lower than the version
+    where the InnoDB system tablespace has been encrypted.
```

Comment:
I think this should be in a version lower than in which we introduced system tablespace encryption? But I'm not sure which version is that.

Also: this would be better in some system tablespace encryption section.

Reply:
Moved to system tablespace. 
16-7 has system tablespace encryption as GA. I checked the RN and could not find when the feature was introduced.